src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2346,9 +2345,9 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public static System.String Format(System.String format, params object[] args) { throw null; }
         public System.CharEnumerator GetEnumerator() { throw null; }
         public override int GetHashCode() { throw null; }
+        public static int GetHashCode(System.ReadOnlySpan<char> value) { throw null; }
```

Comment:
From https://github.com/dotnet/corefx/pull/33316

cc @GrabYourPitchforks, please review for correctness